src/clr-angular/wizard/wizard-stepnav-item.ts

```diff
@@ -26,7 +27,8 @@ import { ClrWizardPage } from './wizard-page';
     '[class.active]': 'isCurrent',
     '[class.disabled]': 'isDisabled',
     '[class.no-click]': '!canNavigate',
-    '[class.complete]': 'isComplete',
+    '[class.complete]': '!hasError && isComplete',
+    '[class.error]': 'hasError && isComplete',
```

Comment:
This conditional or its inverse is being repeated multiple times in this file. It should be made into a method on WizardStepnavItem.

Reply:
Please relocate these repeated conditional checks to a place where they can all live in harmony with one another.